client/src/components/Carousel.jsx

```diff
@@ -5,20 +5,69 @@ let Carousel = (props) => {
 
   // TODO: map over res.data and setState //
   let data = props.books.data
+  let randomImage = "https://source.unsplash.com/random/150x175";
+  let leftArrowState = true;
+  let rightArrowState = false;
+
+  // scroll logic
+  // display only 7 images
+    // left arrow set to "True" by default
+  // if right arrow clicked, remove first 3, show last 3
+    // mark right arrow as "True" and left as "False"
+  // if left arrow clicked, rmeove last 3, show first 3
+    // mark left arrow as "True" and right as "False"
+
+  // default = slice(0, 7).
+    // onClick left = no action
+    // onclick right = re-render w/ slice (3,  10) && set right to false, left to true
+
+  // EVENTUALLY - get arrow to gray out and No-Action if all the way to the right or left
+
   let map = data.map((elem) => {
+    // Title should only be 30 characters long, then "..."
+    let title = '';
+    if (elem.title.length > 38) {
```

Comment:
why 38 and not 30?
maybe you can use const for saving this number?

Reply:
Thanks for catching that typo! Changed.